bin/console

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env php
-<?php declare(strict_types=1);
+<?php
```

Comment:
As `autoload_runtime.php` will include this file again, the `declare(strict_types=1);` would throw an error, because it is not the first statement in the file. (That would be `#!/usr/bin/env php`.) So I omit it for compatibility.